sites/footers/footers-en.html

```diff
@@ -1,48 +1,195 @@
 ---
 {
-	"title": "Content page including complete footer",
+	"title": "Footer",
 	"language": "en",
 	"altLangPage": "footers-fr.html",
 	"secondlevel": false,
-	"dateModified": "2022-05-18",
-	"share": "true"
+	"dateModified": "2022-06-20",
+	"share": "true",
+	"contextualFooter": {
+		"title": "Page-specific contextual footer title",
+		"links": [
+			{ "url": "http://canada.ca/en", "text": "Page-specific Link 1"},
+			{ "url": "http://canada.ca/en", "text": "Page-specific Link 2"},
+			{ "url": "http://canada.ca/en", "text": "Page-specific Link 3"}
+		]
+	}
 }
 ---
-<p>Content page presenting complete default footer including the "About gvernment" section and all utility links from "About this site" section.</p>
-<p>Updated to use the <code>colcount-xs-2</code> class instead of <code>colcount-sm-2</code>.</p>
+<div class="wb-prettify all-pre hide"></div>
+
+<h2>Version 3.0</h2>
+<p>The following changes have been made since version 2.0:</p>
+<ul>
+	<li>Added a "Contextual" section above the "About government" section.
+		<ul>
+			<li>The title and links are defined globally in <code>_config.yml</code>.</li>
+			<li>The title and links can be changed for specific pages in the page's <code>contextualFooter</code> property.</li>
+		</ul>
```

Comment:
Announce these sublist items as being **GCWeb-Jekyll** specific, either by adding a few words to the parent list item after "section. ...", or by putting a prefix on each of the two lines like so: "GCWeb-Jekyll specific: ..."; or something along those lines.